pages_project.go

```diff
@@ -67,18 +67,17 @@ type PagesProjectDeploymentConfigs struct {
 
 // PagesProjectDeploymentConfigEnvironment represents the configuration for preview or production deploys.
 type PagesProjectDeploymentConfigEnvironment struct {
-	EnvVars            map[string]PagesProjectDeploymentVar `json:"env_vars"`
-	CompatibilityDate  string                               `json:"compatibility_date,omitempty"`
-	CompatibilityFlags []string                             `json:"compatibility_flags,omitempty"`
-	KvNamespaces       NamespaceBindingMap                  `json:"kv_namespaces,omitempty"`
-	DoNamespaces       NamespaceBindingMap                  `json:"durable_object_namespaces,omitempty"`
-	D1Databases        D1BindingMap                         `json:"d1_databases,omitempty"`
-	R2Bindings         R2BindingMap                         `json:"r2_buckets,omitempty"`
-}
-
-// PagesProjectDeploymentVar represents a deployment environment variable.
-type PagesProjectDeploymentVar struct {
-	Value string `json:"value"`
+	EnvVars                          EnvironmentVariableMap `json:"env_vars,omitempty"`
+	KvNamespaces                     NamespaceBindingMap    `json:"kv_namespaces,omitempty"`
```

Comment:
similar with these types. as we are breaking changes already, can we fix these up?